enterprise/internal/campaigns/resolvers/changeset_apply_preview_connection.go

```diff
@@ -79,7 +82,115 @@ func (r *changesetApplyPreviewConnectionResolver) Nodes(ctx context.Context) ([]
 	return resolvers, nil
 }
 
-func (r *changesetApplyPreviewConnectionResolver) compute(ctx context.Context) (store.RewirerMappings, *campaigns.Campaign, int, error) {
+type changesetApplyPreviewConnectionStatsResolver struct {
+	push         int32
+	update       int32
+	undraft      int32
+	publish      int32
+	publishDraft int32
+	sync         int32
+	_import      int32
```

Comment:
Not a blocker, but... aesthetically, well, let's just say I won't say "nice." about this `_` here.

You could use plural everywhere?

```suggestion
	imports      int32
```